tests/unit/action-test.js

```diff
@@ -0,0 +1,50 @@
+import Ember from 'ember';
+import action from 'ember-computed-decorators/action';
+import { module, test } from 'qunit';
+
+module('Actions');
+
+test('action decorator works with traditional Ember.Object', function(assert) {
+  assert.expect(2);
+
+  const obj = Ember.Object.extend({
+    @action
+    foo(val) {
+      if (val === 'bar') {
+        assert.ok(true, 'action called through send');
+      } else if (val === 'baz') {
+        assert.ok(true, 'action called directly');
+      }
+    },
+
+    send(fn, ...args) {
```

Comment:
Ah for sure, didn't realize you could dynamically register that easily. Will update them!